Docker postgres upgrade

CelesteNaude opened the pull request:
Creates docker images to upgrade postgres from version 10 to later.
ASG-1737

FritzOnFire commented on postgres/11/Dockerfile:
```diff
@@ -0,0 +1,45 @@
+RUN echo "deb http://ftp.debian.org/debian stretch-backports main" | \
+	tee /etc/apt/sources.list.d/backports.list && \
+	apt-get update --fix-missing && \
+	apt-get install -y postgis postgresql-10-postgis-3 postgresql-11-postgis-3 \
```
You don't need to install postgis 3 for postgresql 10 here. Postgis for version 11 is just fine.

FritzOnFire reviewed (changes requested):
You will need to rebase on your machine so that pgmetrics can be added via gitlfs

CelesteNaude committed:
Docker postgres upgrade

Creates docker images to upgrade postgres from version 10 to newer versions up to 14.
Creates docker images for postgres version 11, 12, 13 and 14.
Version 11 is a copy from version 10.
ASG-1737

FritzOnFire reviewed (approved):
LGTM. (Please merge this, before I forget about it again :P)
